nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml

```diff
@@ -74,9 +65,6 @@ bt_navigator:
       - nav2_assisted_teleop_cancel_bt_node
       - nav2_drive_on_heading_cancel_bt_node
       - nav2_is_battery_charging_condition_bt_node
-    error_code_names:
```

Comment:
Readd. These are for iron, rolling, jazzy and newer. Documentation targets the latest distributions.

Reply:
Sorry I did the pull request just for the first commit and commit other changes for working with Humble. This file and the nav2_map_parameters where not intended to be added to the PR.

Reply:
Please update and I'll take a look again :-)